_search-plugins/concurrent-segment-search.md

```diff
@@ -27,7 +27,7 @@ By default, concurrent segment search is disabled on the cluster. You can enable
 - Cluster level
 - Index level
 
-The index-level setting takes priority over the cluster-level setting. Thus, if the cluster setting is enabled but the index setting is disabled, then concurrent segment search will be disabled for that index.
+The index-level setting takes priority over the cluster-level setting. Thus, if the cluster setting is enabled but the index setting is disabled, then concurrent segment search will be disabled for that index. Because of this, the index-level setting is also not evaluated unless it is explicitly set regardless of the default value configured for the setting. The currently set value of the index-level setting can be retrieved with the index settings API without using the `?include_defaults` parameter.
```

Comment:
```suggestion
The index-level setting takes priority over the cluster-level setting. Thus, if the cluster setting is enabled but the index setting is disabled, then concurrent segment search will be disabled for that index. Because of this, the index-level setting is not evaluated unless it is explicitly set regardless of the default value configured for the setting. You can retrieve the current value of the index-level setting by calling the [Index Settings API]({{site.url}}{{site.baseurl}}/api-reference/index-apis/get-settings/) and omitting the `?include_defaults` query parameter.
```

Reply:
@kolchfa-aws Comma added after "set". Otherwise, LGTM 😄